scripts/set_price_batch.sh

```
@@ -0,0 +1,112 @@
+set_price $TBRG_ADDRESS $TBRG_PRICE "TBRG"
+
+echo ""
+echo "🎉 All prices set successfully!"
+echo ""
+
+# Verify the prices were set correctly
+echo "🔍 Verifying prices..."
+
+verify_price() {
+    local asset_address=$1
+    local expected_price=$2
```

Shellcheck (0.10.0)

[warning] expected_price appears unused. Verify use (or export if used externally).

(SC2034)